tracking/translations_parser/cli/taskcluster.py

```diff
@@ -118,6 +126,14 @@ def main() -> None:
             )
         )
 
+    if args.taskcluster_secret:
+        assert os.environ.get(
+            "TASKCLUSTER_PROXY_URL"
+        ), "When using `--taskcluster-secret`, `TASKCLUSTER_PROXY_URL` environment variable must be set too."
+        secrets = taskcluster.Secrets({"rootUrl": os.environ["TASKCLUSTER_PROXY_URL"]})
+        wandb_secret = secrets.get(args.taskcluster_secret)
+        os.environ.setdefault("WANDB_API_KEY", wandb_secret["secret"])
```

Comment:
the yaml inside the `project/translations/level-1/weights-and-biases` is `token: <w_&_b_token>`

Reply:
Fixed in 06e3d6fc7945a58897918be28acb49bf109242e9, thx for the review